checkov/github_actions/runner.py

```diff
@@ -21,5 +24,32 @@ def import_registry(self):
         return self.block_type_registries['jobs']
 
     def _parse_file(self, f):
-        if ".github/workflows/" in os.path.abspath(f):
+        if self.is_workflow_file(f):
             return super()._parse_file(f)
+
+    def is_workflow_file(self, f):
+        abspath = os.path.abspath(f)
+        return WORKFLOW_DIRECTORY in abspath and (abspath.endswith("yml") or abspath.endswith("yaml"))
+
+    def get_images(self, f):
+        # workflow file can have a job run within a container.
+        # example: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-credentials-for-a-container-registry
+
+        images = []
+
+        workflow, workflow_line_numbers = self._parse_file(f)
+        jobs = workflow.get("jobs", {})
+        for job_name, job_object in jobs.items():
+            if isinstance(job_object, dict):
+                container = job_object.get("container", {})
+                image = None
+                if isinstance(container, dict):
+                    image = container.get("image", "")
```

Comment:
```suggestion
                    image = container.get("image")
```
then it will be `None` as above no need to change it to an empty string